metricsaggregator/src/test/java/io/dockstore/metricsaggregator/client/cli/MetricsAggregatorClientIT.java

```diff
@@ -136,6 +137,7 @@ public static void afterClass() {
         SUPPORT.after();
     }
 
+    @Disabled(value = "Disabled because localstack does not support Athena in the free version")
```

Comment:
Can also provide link for future investigation if situation may change 
(or if we do purchase a subscription)